### PR TITLE
Python: added class FieldProperties.

### PR DESCRIPTION
**Pull request by stefoss23**
When cmake switch ENABLE_3DPROPS_TESTING is ON, this will enable testing and use of FieldPropsManager in Python as well.

### CODE REVIEW

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
F....g good!

File: python/cxx/field_props.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#include <pybind11/stl.h>
+
+#include "export.hpp"
+#include "converters.hpp"
+
+namespace {
+
+    py::list getitem( const FieldPropsManager& m, const std::string& kw) {
+        if (m.has<int>(kw))
```
Comment (joakim-hove) [inline python/cxx/field_props.cpp]: This must return a numpy array.

**Comment by stefoss23** [comment]
jenkins build this please

**Pull request merged**